docs/solution-pack-guide.md

```diff
@@ -0,0 +1,27 @@
+# Usage of SOC Simulator Solution Pack
+
+## Configuring the FortiSOAR SOC Simulator Connector
+
+This solution pack will install and pre-configure the FortiSOAR SOC Simulator Connector by default. 
+
+## Scenario Module
```

Comment:
What is purpose of this heading here?

Reply:
We also nee to mention about functionalities provided by this SP like, operations - simulate, reset, etc. and then explain them one by one as below;

Reply:
Created separate sections for 'Simulate' and 'Reset' scenarios. Please review